spec/system/provider_interface/see_individual_application_spec.rb

```diff
@@ -5,12 +5,11 @@
   include DfESignInHelpers
 
   before do
-    TestSuiteTimeMachine.travel_permanently_to(2021, 3, 1, 12, 0, 0)
+    TestSuiteTimeMachine.travel_permanently_to(2022, 10, 15, 12, 0, 0)
```

Comment:
once the candidate PR is merged, this can use the new helper `travel_permanently_to`.

also should we introduce a variable that explains the significance of this date?

Reply:
I should update the docs.  `travel_permanently_to` has been wrapped in the `time:` option so we can tighten it up even more.